.github/workflows/release.yml

```diff
@@ -132,33 +135,25 @@ jobs:
           export GH_PAGES_DIR="$PWD/gh-pages"
           ./build/dev/publish-plugin-registry-to-gh-pages.sh
 
-          cd ${{ env.GITHUB_WORKSPACE }}
-          message=$(jq -r '.text' mattermost.json)
-          echo "$( jq '.text = "'$message'. publish to GH succeeded."' mattermost.json )" > mattermost.json   
+          echo MM_MESSAGE_APPEND='${{ env.MM_MESSAGE_APPEND }} Published package to npmjs https://www.npmjs.com/package/@eclipse-che/plugin-registry-generator/v/${{ github.event.inputs.version }}.' >> $GITHUB_ENV
       - name: publish to npmjs
         if: ${{ github.event.inputs.publishToNpmJs == 'true' }}
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
         run: |
-          cd ${{ env.GITHUB_WORKSPACE }}
+          cd ${GITHUB_WORKSPACE}
```

Comment:
do we need this cd? in all other `run` sections you don't move into the workspace since I assume you're already there?

Reply:
it wouldn't hurt to be there